libmamba/tests/src/solver/test_problems_graph.cpp

```diff
@@ -33,56 +33,53 @@
 using namespace mamba;
 using namespace mamba::solver;
 
-namespace
+TEST_CASE("symmetric")
 {
-    TEST_CASE("symmetric")
-    {
-        auto c = conflict_map<std::size_t>();
-        REQUIRE(c.size() == 0);
-        REQUIRE_FALSE(c.has_conflict(0));
-        REQUIRE_FALSE(c.in_conflict(0, 1));
-        REQUIRE(c.add(0, 1));
-        REQUIRE(c.add(1, 2));
-        REQUIRE_FALSE(c.add(1, 2));
-        REQUIRE(c.has_conflict(0));
-        REQUIRE(c.in_conflict(0, 1));
-        REQUIRE(c.in_conflict(1, 2));
-        REQUIRE(c.has_conflict(2));
-        REQUIRE_FALSE(c.in_conflict(0, 2));
-        // With same
-        REQUIRE(c.add(5, 5));
-        REQUIRE(c.has_conflict(5));
-        REQUIRE(c.in_conflict(5, 5));
-    }
+    auto c = conflict_map<std::size_t>();
+    REQUIRE(c.size() == 0);
+    REQUIRE_FALSE(c.has_conflict(0));
+    REQUIRE_FALSE(c.in_conflict(0, 1));
+    REQUIRE(c.add(0, 1));
+    REQUIRE(c.add(1, 2));
+    REQUIRE_FALSE(c.add(1, 2));
+    REQUIRE(c.has_conflict(0));
+    REQUIRE(c.in_conflict(0, 1));
+    REQUIRE(c.in_conflict(1, 2));
+    REQUIRE(c.has_conflict(2));
+    REQUIRE_FALSE(c.in_conflict(0, 2));
+    // With same
+    REQUIRE(c.add(5, 5));
+    REQUIRE(c.has_conflict(5));
+    REQUIRE(c.in_conflict(5, 5));
+}
 
-    TEST_CASE("remove")
-    {
-        auto c = conflict_map<std::size_t>({ { 1, 1 }, { 1, 2 }, { 1, 3 }, { 2, 4 } });
-        REQUIRE(c.size() == 4);
-
-        REQUIRE(c.in_conflict(2, 4));
-        REQUIRE(c.in_conflict(4, 2));
-        REQUIRE(c.remove(2, 4));
-        REQUIRE_FALSE(c.in_conflict(4, 2));
-        REQUIRE_FALSE(c.in_conflict(2, 4));
-        REQUIRE(c.has_conflict(2));
-        REQUIRE_FALSE(c.has_conflict(4));
-
-        REQUIRE_FALSE(c.remove(2, 4));
-
-        REQUIRE(c.remove(1));
-        REQUIRE_FALSE(c.has_conflict(1));
-        REQUIRE_FALSE(c.in_conflict(1, 1));
-        REQUIRE_FALSE(c.in_conflict(1, 2));
-        REQUIRE_FALSE(c.in_conflict(3, 1));
-    }
+TEST_CASE("remove")
+{
+    auto c = conflict_map<std::size_t>({ { 1, 1 }, { 1, 2 }, { 1, 3 }, { 2, 4 } });
+    REQUIRE(c.size() == 4);
+
+    REQUIRE(c.in_conflict(2, 4));
+    REQUIRE(c.in_conflict(4, 2));
+    REQUIRE(c.remove(2, 4));
+    REQUIRE_FALSE(c.in_conflict(4, 2));
+    REQUIRE_FALSE(c.in_conflict(2, 4));
+    REQUIRE(c.has_conflict(2));
+    REQUIRE_FALSE(c.has_conflict(4));
+
+    REQUIRE_FALSE(c.remove(2, 4));
+
+    REQUIRE(c.remove(1));
+    REQUIRE_FALSE(c.has_conflict(1));
+    REQUIRE_FALSE(c.in_conflict(1, 1));
+    REQUIRE_FALSE(c.in_conflict(1, 2));
+    REQUIRE_FALSE(c.in_conflict(3, 1));
 }
 
+using namespace mamba::specs::match_spec_literals;
+using Request = solver::Request;
+
 namespace
```

Comment:
I added this one because `mpkg` function is also present in another test file (and I added anon namespace there as well)